project-service/gulpfile.babel.js

```diff
@@ -0,0 +1,21 @@
+import path from 'path';
+import gulp from 'gulp';
+import shell from 'gulp-shell';
```

Comment:
I'm using tap-spec here in the shell script...will that actually work from root?